seatunnel-transforms-v2/src/test/java/org/apache/seatunnel/transform/replace/ReplaceTransformTest.java

```diff
@@ -0,0 +1,115 @@
+import org.apache.seatunnel.api.table.type.BasicType;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class ReplaceTransformTest {
+
+    private static final CatalogTable DEFAULT_TABLE =
+            CatalogTable.of(
+                    TableIdentifier.of("test", "Database-x", "Schema-x", "Table-x"),
+                    TableSchema.builder()
+                            .column(
+                                    PhysicalColumn.of(
+                                            "f1",
+                                            BasicType.LONG_TYPE,
+                                            null,
+                                            null,
+                                            false,
+                                            null,
+                                            null))
+                            .column(
+                                    PhysicalColumn.of(
+                                            "f2",
+                                            BasicType.LONG_TYPE,
+                                            null,
+                                            null,
+                                            true,
+                                            null,
+                                            null))
+                            .column(
+                                    PhysicalColumn.of(
+                                            "f3",
+                                            BasicType.LONG_TYPE,
+                                            null,
+                                            null,
+                                            true,
+                                            null,
+                                            null))
+                            .primaryKey(PrimaryKey.of("pk1", Arrays.asList("f1")))
+                            .constraintKey(
+                                    ConstraintKey.of(
+                                            ConstraintKey.ConstraintType.UNIQUE_KEY,
+                                            "uk1",
+                                            Arrays.asList(
+                                                    ConstraintKey.ConstraintKeyColumn.of(
+                                                            "f2", ConstraintKey.ColumnSortType.ASC),
+                                                    ConstraintKey.ConstraintKeyColumn.of(
+                                                            "f3",
+                                                            ConstraintKey.ColumnSortType.ASC))))
+                            .build(),
+                    Collections.emptyMap(),
+                    Collections.singletonList("f2"),
+                    null);
+
+    @Test
+    public void testRepalceTransformSchangeChange() {
```

Comment:
Could you add some test case to make sure the transform fine after schema changed? For example, still can replace data as expected.